core/pom.xml

```diff
@@ -18,7 +18,13 @@
         <dependency>
             <groupId>org.camunda.bpm</groupId>
             <artifactId>camunda-engine</artifactId>
-            <version>RELEASE</version>
```

Comment:
We actually intentionally always compile against RELEASE, but test via Travis CI against the many versions *provided* by user projects back to 7.2.0. (For the same reason, the tests are in a separate module.)